resources/js/components/GraphqlMutation.vue

```diff
@@ -105,7 +116,7 @@ export default {
     },
 
     methods: {
-        async mutate() {
+        async mutateFn() {
```

Comment:
This might cause problems with 
https://docs.rapidez.io/3.x/graphql-components.html#:~:text=partial%2Dsubmit%20must%20always%20be%20on%20the%20direct%20child%20of%20a%20Vue%20component%20and%20have%20its%20original%20function%20name.

Could you test to make sure this causes no problems with partials-submit?

Reply:
Yes, this works fine in this case because the actual underlying function name within `__vue__` stays the same. From what I can tell the issue that happens in the case shown in the docs is that the name within the render itself is different, where then the `submitPartials` function looks for that function name within the vue component and comes up blank.